diff --git a/dist/app/composables/asyncData.d.ts b/dist/app/composables/asyncData.d.ts
index 5c4d39f9bacd976de73407bca02b5296d57f80cf..2e69bdad15f794f17ad8bd49863b5a02e17744de 100644
--- a/dist/app/composables/asyncData.d.ts
+++ b/dist/app/composables/asyncData.d.ts
@@ -27,7 +27,7 @@ export interface AsyncDataOptions<ResT, DataT = ResT, PickKeys extends KeysOf<Da
      * A `null` or `undefined` return value will trigger a fetch.
      * Default is `key => nuxt.isHydrating ? nuxt.payload.data[key] : nuxt.static.data[key]` which only caches data when payloadExtraction is enabled.
      */
-    getCachedData?: (key: string, nuxtApp: NuxtApp) => DataT;
+    getCachedData?: (key: string, nuxtApp: NuxtApp) => NoInfer<DataT>;
     /**
      * A function that can be used to alter handler function result after resolving.
      * Do not use it along with the `pick` option.
